python/tests/test_rate_limit_keys.py

```
@@ -0,0 +1,237 @@
+
+@contextmanager
+def trusted_proxies(*cidrs: str):
+    settings.BOLT_TRUSTED_PROXIES = list(cidrs)
+    try:
+        yield
+    finally:
+        del settings.BOLT_TRUSTED_PROXIES
+
+
+@pytest.fixture(autouse=True)
+def _fresh_limiters():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fresh_limiters`

(ANN202)